changes/e25248c44d5ba33a43ae937a0bb25fd5.yaml

```diff
@@ -0,0 +1,6 @@
+---
+desc: Fixed an issue where certain exceptions raised while calling a function in Storm
+    were not handled correctly.
```

Comment:
Maybe `were not providing appropriate details about the origin of the exception` or something a hair more specific?

Reply:
Yeah these were initally python TypeError exceptions making it through which was what the "not handled correctly" was referring to. I can tweak it